azure/appservice-windows/main.tf

```diff
@@ -0,0 +1,89 @@
+resource "azurerm_resource_group" "app" {
+  name     = "rg-${var.project}-${var.environment}"
+  location = local.location
+
+  tags = {
+    country     = var.country
+    environment = var.environment
+    squad       = var.squad
+    product     = var.product
+  }
+}
+
+resource "azurerm_app_service_plan" "app" {
+  name                = "asp-${var.project}-${var.environment}"
+  location            = azurerm_resource_group.app.location
+  resource_group_name = azurerm_resource_group.app.name
+  kind                = "Windows"
+  reserved            = false
+
+  sku {
+    tier = var.plan
+    size = var.size
+  }
+
+  tags = {
+    country     = var.country
+    environment = var.environment
+    squad       = var.squad
+    product     = var.product
+  }
+}
+
+resource "azurerm_app_service" "app" {
+  name                    = "as-${var.project}-${var.environment}"
+  location                = azurerm_resource_group.app.location
+  resource_group_name     = azurerm_resource_group.app.name
+  app_service_plan_id     = azurerm_app_service_plan.app.id
+  https_only              = true
+  enable_client_affinity  = true
+  stack                   = var.stack
+  identity {
+    type = "SystemAssigned"
+  }
+  site_config = {
+    always_on                 = true
+    dotnet_framework_version  = var.dotnet_framework_version
+    ftps_state                = "All allowed"
+    managed_pipeline_mode     = "Integrated"
+    use_32_bit_worker_process = false
+    websockets_enabled        = false
+    remote_debugging_enabled  = false
+  }
+  tags = {
```

Comment:
The tags I believe can be removed, since the tags will be inherited from the RG.

Reply:
done